components/dictionary.txt

```diff
@@ -305,6 +310,10 @@ Satija
 Sca
 scater
 SCE
+ScPCA
+SCPCL
+SCPCP
+SCPCS
```

Comment:
Were these actually showing up as errors, or just prophylactic?

Reply:
They were showing up as spelling errors - 
[spell_check_errors.tsv.txt](https://github.com/AlexsLemonade/training-modules/files/10125604/spell_check_errors.tsv.txt)

Reply:
Huh, I would have expected combinations of numbers and letters like `SCPCP000005` to be skipped by the spellchecker. I don't really feel strongly about this, but we might want to use code formatting for these IDs, which would also prevent the spelling errors.